library/lib.sh

```diff
@@ -84,7 +85,7 @@ rolesHandleVault() {
     local playbook_file=$role_path/tests/$2
     local vault_pwd_file=$role_path/tests/vault_pwd
     local vault_variables_file=$role_path/tests/vars/vault-variables.yml
-    local no_vault_file=$role_path/no-vault-variables.txt
+    local no_vault_file=$role_path/tests/no-vault-variables.txt
```

Comment:
the directory where to find the vault files depends on if testing a legacy role or a collection - e.g. /usr/share/ansible/roles/linux-system-roles.$role/tests vs. /usr/share/ansible/collections/ansible-collections/fedora/linux_system_roles/tests/$role/

Reply:
This test does all the preparation on the legacy role, and then converts the role to collection, and tests the collection. We might want to revision it when we want to test collections right away. Maybe when we get downstream tests use this test suite, for now it is fine as is I'd say.